### PR TITLE
Re-apply qs backpressure increase with buffer latency increase

### PR DESCRIPTION
**Pull request by igor-aptos**
## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by vusirikala — APPROVED** [review]
As graceful overload test failed last time, I ran graceful overload test here. It seems to be working.
https://github.com/aptos-labs/aptos-core/actions/runs/9865912053

File: config/src/config/mempool_config.rs [hunk 1]
```diff
@@ -85,8 +85,8 @@ impl Default for MempoolConfig {
             system_transaction_timeout_secs: 600,
             system_transaction_gc_interval_ms: 60_000,
             broadcast_buckets: DEFAULT_BUCKETS.to_vec(),
-            eager_expire_threshold_ms: Some(10_000),
-            eager_expire_time_ms: 3_000,
+            eager_expire_threshold_ms: Some(15_000),
```
Comment (sitalkedia) [inline config/src/config/mempool_config.rs:88]: Can you explain the downside of increasing this?

Comment (igor-aptos) [reply config/src/config/mempool_config.rs:88]: new constants make the logic like this:

if there is any transaction in the mempool (that never entered parking lot) that was there for longer than eager_expire_threshold_ms (which is increased from 10s -> 15s here) - that means we have a significant backlog.

if that is the case, we pull into batches only transactions that have at least eager_expire_time_ms time left (increased from 3s to 6s)


We've increased both, as with reduced QS backpressure, we see it taking more time from batch creation to batch being included in the block.

So the side-consequence of this change is that - if that during backlog, if you use expiration <6s, your transactions will be ignored (previously <3s).

Comment (sitalkedia) [reply config/src/config/mempool_config.rs:88]: Sounds good to me. Slightly orthogonal but do we really need this logic after @vusirikala 's change to exclude expired transaction in QS backpressure calculation in https://github.com/aptos-labs/aptos-core/pull/13850/files. cc - @bchocho

Comment (igor-aptos) [reply config/src/config/mempool_config.rs:88]: creating useless batches, and then requiring everyone to fetch them, just in order to throw them out - probably has negative effect on our overall throughput.

**Review by sitalkedia — APPROVED** [review]
Let's revert the forge changes before landing.

File: config/src/config/mempool_config.rs (hunk 1 above)
Comment (vusirikala) [reply config/src/config/mempool_config.rs:88]: @sital This is the run for graceful overload test with my changes and increasing QS backpressure limits without changing the `eager_expire_threshold_ms` config.
https://github.com/aptos-labs/aptos-core/pull/13964#issuecomment-2221167236

**Pull request merged**